sql-cli/sql_cli/dag_generator.py

```diff
@@ -103,15 +103,16 @@ def generate_dag(directory: Path, dags_directory: Path, generate_tasks: bool) ->
         start_date=datetime(2020, 1, 1),
         sql_files=sql_files,
     )
-    template_dag_name = "gen_tasks_dag"
     if generate_tasks:
+        template_file = Path("templates/gen_tasks_dag.py.jinja2")
```

Comment:
How about we define a constant `TEMPLATES_DIRECTORY = Path("templates")` and set here `templates_file = TEMPLATES_DIRECTORY / "gen_tasks_dag.py.jinja2"`